analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/SolutionBuilder.cs

```diff
@@ -44,8 +44,8 @@ private SolutionBuilder(Solution solution)
             Solution = solution;
         }
 
-        public ProjectBuilder AddTestProject(AnalyzerLanguage language, bool createExtraEmptyFile = true) =>
-            AddProject(language, $"{GeneratedAssemblyName}{ProjectIds.Count}.Tests", createExtraEmptyFile);
+        public ProjectBuilder AddTestProject(AnalyzerLanguage language, bool createExtraEmptyFile = true, OutputKind outputKind = OutputKind.DynamicallyLinkedLibrary) =>
```

Comment:
Where is this used? VS doesn't find me any invocation with the new argument.

Reply:
nice catch. Initially I was using `.AddTestProject(AnalyzerLanguage.FromPath(path), OutputKind.Console..)` but then I had to change and forgot to revert this.